pkgs/top-level/python-packages.nix

```diff
@@ -17265,4 +17265,30 @@ let
     };
   };
 
+  dyn = pkgs.pythonPackages.buildPythonPackage rec {
```

Comment:
`pkgs.pythonPackages.` is not necessary. The order in this file is also mostly alphabetically. Could you put this in a better place?